src/main/java/net/mcreator/ui/modgui/BiomeGUI.java

```diff
@@ -87,6 +87,7 @@
 
 	private final JColor airColor = new JColor(mcreator, true);
 	private final JColor grassColor = new JColor(mcreator, true);
+	private final JColor foliageColor = new JColor(mcreator, true);

```

Comment:
Add test value for this similar to this: https://github.com/MCreator/MCreator/blob/5fa77237a5f1512baf9d1f42dab8cd92a815a998/src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java#L213